src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "country")
 public class Country implements Cloneable {
```

Comment:
The `Cloneable` interface and the corresponding `clone` method implementation should be removed. Hibernate entities are not meant to be cloned in this manner, and it could lead to unexpected behavior, especially with managed entities and proxy objects.